less/navbar.less

```diff
@@ -130,6 +130,12 @@
   z-index: @zindex-navbar-fixed;
   backface-visibility: hidden;
 
+  .navbar-nav {
+    > li > a {
```

Comment:
Only thing is it would be nice if the background change was nearby, rather than another part of the file.  From a maintainability perspective, it's hard to know if the colour change happens based on the same selector or a different one (in which case it could be buggy).  And should we wish to change this in future they would likely change together.